### ISSUE TITLE
Layout/IndentationWidth inconsistency between Ruby 3.4 (Prism) and Ruby 3.3 (Legacy) for chained method blocks

### ISSUE THREAD

**Issue by keithrbennett** [rubocop/rubocop#14746]
I have encountered a regression/inconsistency in Layout/IndentationWidth when running RuboCop 1.81.7 on Ruby 3.4 (Prism parser) compared to Ruby 3.3.

  When using a method chain that ends with a block, RuboCop on Ruby 3.4 accepts code where the block body is aligned with the method call (0 spaces of indentation relative to the method),
  whereas RuboCop on Ruby 3.3 enforces standard indentation (2 spaces relative to the method).

  This causes code formatted and accepted locally (Ruby 3.4) to fail in CI (Ruby 3.3).

  Reproduction:

  repro.rb:
```ruby
    1 # frozen_string_literal: true
    2 
    3 def test_indentation
    4   output = 'some output'
    5   output.to_s
    6     .lines
    7     .each do |line|
    8     puts line # Ruby 3.4 accepts this (aligned with .each)
    9   end
   10 end
```

  .rubocop.yml:
```ruby
   1 Layout/IndentationWidth:
   2   Enabled: true
   3   Width: 2
```

  Observed Behavior:

   * Ruby 3.4.1 + RuboCop 1.81.7: No Layout/IndentationWidth offenses.
   * Ruby 3.3.6 + RuboCop 1.81.7:

```
   1     repro.rb:7:5: C: [Correctable] Layout/IndentationWidth: Use 2 (not 0) spaces for indentation.
   2         puts line
   3         ^^^^
```

  Expected Behavior:

  RuboCop should report offenses consistently across Ruby versions, preferably enforcing the standard 2-space indentation relative to the block start in both cases.

  Context:

   * RuboCop version: 1.81.7
   * Ruby version: 3.4.1 (Prism default) vs 3.3.x
   * OS: Linux / GitHub Actions Ubuntu runner

**Comment by koic** [comment]
This issue appears to have been resolved by https://github.com/rubocop/rubocop/pull/14713. Please upgrade to the latest RuboCop 1.82.1.

**Comment by keithrbennett** [comment]
Thank you, @koic. I've confirmed on my end that the issue is fixed.

My apologies for not thinking of upgrading before posting the issue.
